storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/AnonymousCredentials.java

```diff
@@ -47,7 +47,7 @@ public RequestPolicy create(RequestPolicy nextRequestPolicy, RequestPolicyOption
      * <a href=https://docs.microsoft.com/en-us/azure/storage/blobs/storage-manage-access-to-resources>Manage Access to Storage Resources</a>
      * <a href=https://docs.microsoft.com/en-us/rest/api/storageservices/set-container-acl>Set Container Permissions</a>
      */
-    private final class AnonymousCredentialsPolicy implements RequestPolicy {
+    private final static class AnonymousCredentialsPolicy implements RequestPolicy {
```

Comment:
Why did we make this static?
Does this change impact, how this class is used in the code?